app/Controller/PagesController.php

```diff
@@ -72,5 +118,59 @@ public function display() {
 			}
 			throw new NotFoundException();
 		}
-	}
+  }
+
+  private $顧客名;
+
+  function set顧客名($v)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Avoid variables with short names like $v. Configured minimum length is 3.](https://www.codacy.com/app/actcat/sideci_php_sandbox/file/1229144208/issues/source?bid=2928887&fileBranchId=2932683#l125)